src/scss/atlas-theme/_atlas.scss

```diff
@@ -1,5 +1,5 @@
 @import "alerts";
-@import "breadcrumbs";
+@import "badges";
```

Comment:
Hey @natecavanaugh, yeah thats a mistake, it must have happened when I rebased. I update d the pull with the correction. Sorry about that.